app/controllers/studios_controller.rb

```diff
@@ -0,0 +1,29 @@
+class StudiosController < ApplicationController
+  def index
+    @studios = Studio.all
+  end
+
+  def show
+    @studio = Studio.find(params[:id])
+  end
+
+  def new
+    @studio = Studio.new
+  end
+
+  def create
+    @studio = Studio.new(studio_params)
+    @studio.user = current_user
+    if @studio.save
+      redirect_to "/studios/#{@studio.id}"
+    else
+      render :new
+    end
+  end
+
+  private
+  def studio_params
+    # params[:studio][:user] = current_user
+    params.require(:studio).permit(:name, :address, :description, :price, :photo, :user)
+  end
+end
```

Comment:
We have a search bar on our home page that will most likely route to `Studio Index`, yet this controller isn't set up to handle the `index` action. I think we should focus on making one action/user journey work at a time so that we make sure it is completing the necessary function.

Reply:
Sure. Now the controller gives @studios to the view. What else do you need?

Reply:
Also, it might be better if the person working on the navbar takes care of that. That doesn't necessarily belong to this branch. And this way, we can merge the branch and move on.